.github/workflows/pr-cpu.yaml

```diff
@@ -9,36 +9,39 @@ concurrency:
   cancel-in-progress: ${{ github.ref != 'refs/heads/main' && github.ref != 'refs/heads/dev' }}
 jobs:
   pytest-cpu:
-    uses: mosaicml/ci-testing/.github/workflows/pytest-cpu.yaml@v0.0.9
+    name: ${{ matrix.name }}
+    runs-on: linux-ubuntu-latest
+    if: github.repository_owner == 'mosaicml'
```

Comment:
Do we need this if statement here
```
if: github.repository_owner == 'mosaicml'
```

Reply:
Yes, this is so forks don't run this code as well.